Location component

* File location: components/common/ folder
* Props: coordinates, lat & lon
* Output, see UI below (in the red rectangle):
<img width="1440" alt="Screen Shot 2021-09-27 at 11 54 13 AM" src="https://user-images.githubusercontent.com/5744708/134843226-e5bb7cb1-1990-4b65-8e0d-a125d6fe21d8.png">
* API: will use the API created by #153 please mock the API if it is not ready yet.


---
Some hints for building components:
* Design resource: https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap?node-id=2497%3A11584
* A tutorial of how to use the Cypress component tool to build components: 
  https://github.com/Greenstand/treetracker-wallet-web#how-to-build-a-component
* We are using MaterialUI to build the app: https://material-ui.com/
* When you are ready to raise PR for the component, please attach the screenshot of the component for review.
* Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
* Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
* To know more about our organization, visit our [website](https://greenstand.org).

@dadiorchen I'm currently using the latest stable version of ubuntu also after deleting the lock file I get this error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: client@1.14.0
npm ERR! Found: eslint@6.6.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^6.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^5.16.0 || ^6.8.0 || ^7.2.0" from eslint-config-airbnb-base@14.2.0
npm ERR! node_modules/eslint-config-airbnb-base
npm ERR!   dev eslint-config-airbnb-base@"^14.2.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /root/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-09-27T10_10_17_305Z-debug.log
```

**Describe the bug**
The project crashes when we execute npm install on Ubuntu 20.04.3 LTS .

**To Reproduce**
Steps to reproduce the behavior:
(we recommend record a screenshot to show the case, like using http://loom.com to record)


**Expected behavior**
Should install all the dependencies but gives this error:
```
npm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile 
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile 
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin"} (current: {"os":"linux","arch":"x64"})
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  undefined
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-09-28T04_53_28_997Z-debug.log
```
After deleting the package-lock.json and re-executing npm install:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: client@1.14.0
npm ERR! Found: eslint@6.6.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^6.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^5.16.0 || ^6.8.0 || ^7.2.0" from eslint-config-airbnb-base@14.2.0
npm ERR! node_modules/eslint-config-airbnb-base
npm ERR!   dev eslint-config-airbnb-base@"^14.2.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/ozer/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ozer/.npm/_logs/2021-09-28T04_56_20_898Z-debug.log
```



** Node.js version
v16.9.1

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.3 LTS

--- a/src/common/Location.cy.js
+++ b/src/common/Location.cy.js
@@ -1,0 +1,13 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import Location from './Location';
+
+const info = {
+  entityLocation: 'Shirimatunda,Tanzania',
+};
+
+describe('Location', () => {
+  it('Location', () => {
+    mount(<Location {...info} />);
+  });
+});

--- a/src/common/Location.js
+++ b/src/common/Location.js
@@ -1,0 +1,47 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Box } from '@material-ui/core';
+import LocationOnOutlinedIcon from '@material-ui/icons/LocationOnOutlined';
+
+ 
+const useStyles = makeStyles((theme) => ({
+
+
+  container: {
+    position: 'absolute',
+    zIndex: 1,
+    justifyContent:"space-between",      
+  },
+
+  entityLocation: {
+    fontWeight: 500,
+    color: '#bebcbc',
+    fontSize: 15,
+    textAlign:"center",
+    display:"flex",   
+  },
+  LocationOnOutlinedIcon: {
+    padding: '0px 5px',
+    fontSize: 15,  
+    justifyContent:"space-between", 
+  },
+}));
+
+function Location({
+  entityLocation,
+}) {
+  const classes = useStyles();
+  return (
+    <Box
+      className={classes.container}
+    >
+      <div>
+          <div className={classes.entityLocation}> <LocationOnOutlinedIcon  className={classes.LocationOnOutlinedIcon}/>{entityLocation}</div>      
+      </div>
+ 
+    </Box>
+  );
+}
+
+export default Location;
+

